### ISSUE TITLE
Day 5

### ISSUE THREAD

**Issue by arpit-dwivedi** [Learn-Write-Repeat/Learn-Write-Repeat.github.io#5]
Make a template for technical articles, with the bootstrap studio, using all advance options. In the template put content from your files in [open-contributions](https://github.com/Learn-Write-Repeat/Open-contributions) repository, both markdown, and jupyter part. The page should contain all the information from both the files and also a link to download the files.

Incase needed:

1. **[Markdowntohtml](https://markdowntohtml.com/) :** For converting markdown files to HTML.
2. **[nbconverter](https://pypi.org/project/nbconvert/) :** For converting jupyter notebook to HTML.

> Use these convert your file in HTML, do not just convert both the files and merge it, make the whole page interactive.

After doing this, zip the files and attach below with your comments on today's summary.

**Comment by akshayadme** [comment]
- Today's task was to design template for technical article using bootstrap studio.

- I have created this web design by including navbar, header, footer and some content about python topics which is properly responsive for every device.

- I have attached the file of template in zip format.

[Python.zip](https://github.com/Learn-Write-Repeat/Learn-Write-Repeat.github.io/files/5284492/Python.zip)

**Comment by reddymahendra52** [comment]
* Today's task was to design a template for a technical documentation page using bootstrap studio.
* Since it is a technical documentation page I tried to keep it as simple as possible so that it can be easily understood and accessible
* My template in a whole has two section one section contains the documentation contributor's information with the list of topics and the other section contains the actual info.
* In the template with content I have just filled the starting topics so that it gives a basic idea how it will look after it is completed fully.


[Template.zip](https://github.com/Learn-Write-Repeat/Learn-Write-Repeat.github.io/files/5284186/Template.zip)
[Template with content.zip](https://github.com/Learn-Write-Repeat/Learn-Write-Repeat.github.io/files/5284172/Template.with.content.zip)

**Comment by Ansuman9020** [comment]
- Today's task was to design template for technical article using bootstrap studio.
- I created a page containing navbar, articles section and the footer.
- It has been linked to other page which views the documentation of the contributed tasks.


[Articles.zip](https://github.com/Learn-Write-Repeat/Learn-Write-Repeat.github.io/files/5284268/Articles.zip)

**Comment by manvi-123894** [comment]
- Today's task was to create a template for a technical blog website using the bootstrap studio.

- I have created a template including a navbar, content section and a footer.

[Technical_Blog_Template.zip](https://github.com/Learn-Write-Repeat/Learn-Write-Repeat.github.io/files/5284350/Technical_Blog_Template.zip)

**Comment by chirag0821** [comment]
- Today's task was to design a template for technical article.
- I used the Clean Blog template and added all the content of my .ipynb and .md file contributions.
- I tried my best to make it fully responsive, I added my own media query to make it responsive.
- After that I added all the advance SEO features and footer to the template.
[Template zip](https://drive.google.com/u/0/uc?id=1L0atGKrq3vzesa7_YKRGqCEzzIp72tzf&export=download)
[Template with content zip](https://drive.google.com/u/0/uc?id=1m8h43Od1whS-KnlcX5UGf9oKf1ZOSj4I&export=download)
